src/CommunityToolkit.Maui.Core/Platform/StatusBar/StatusBar.android.cs

```diff
@@ -31,35 +30,26 @@ static void PlatformSetStyle(StatusBarStyle style)
 		switch (style)
 		{
 			case StatusBarStyle.DarkContent:
-				AddBarAppearanceFlag(Activity, (StatusBarVisibility)SystemUiFlags.LightStatusBar);
+				SetStatusBarAppearance(Activity, true);
 				break;
 
 			case StatusBarStyle.Default:
 			case StatusBarStyle.LightContent:
-				RemoveBarAppearanceFlag(Activity, (StatusBarVisibility)SystemUiFlags.LightStatusBar);
+				SetStatusBarAppearance(Activity, false);
 				break;
 
 			default:
 				throw new NotSupportedException($"{nameof(StatusBarStyle)} {style} is not yet supported on Android");
 		}
 	}
 
-	static void AddBarAppearanceFlag(Activity activity, StatusBarVisibility flag) =>
-		SetBarAppearance(activity, barAppearance => barAppearance |= flag);
-
-
-	static void RemoveBarAppearanceFlag(Activity activity, StatusBarVisibility flag) =>
-		SetBarAppearance(activity, barAppearance => barAppearance &= ~flag);
-
-
-	static void SetBarAppearance(Activity activity, Func<StatusBarVisibility, StatusBarVisibility> updateAppearance)
+	static void SetStatusBarAppearance(Activity activity, bool lightStatusBars)
```

Comment:
Since this is a boolean, let's add the `is` prefix. Remember to change other references as well.
```suggestion
	static void SetStatusBarAppearance(Activity activity, bool isLightStatusBars)
```